api/v1alpha1/bundle_types.go

```diff
@@ -80,6 +80,11 @@ type GitSource struct {
 	// is required. Setting more than one field or zero fields will result in an
 	// error.
 	Ref GitRef `json:"ref"`
+	// Secret contains the secret name that has authorization information for HTTPS protocol and is in the namespace that the provisioner is deployed.
+	// The secret contains the user ID (key: user) and the personal access token to the repository (key: token)
```

Comment:
Maybe make this slightly more concise and direct?
```suggestion
	// Secret contains the secret name that has authorization information for HTTPS protocol and is in the namespace that the provisioner is deployed.
	// The secret is expected to contain `data.username` and `data.password` for the username and password, respectively.
```

Reply:
Thanks!  I'll update `token` and `personal access token`.  It's a good practice to use the personal access token instead of the password.